examples/phi2/phi2.py

```diff
@@ -124,55 +132,64 @@ def main(raw_args=None):
         legacy_optimization_setting(template_json)
 
     # add pass flows
-    model_type = str(args.model_type)
-    template_json["pass_flows"] = SUPPORTED_WORKFLOWS[model_type]
-    if args.optimum_optimization:
-        # if args.model_type in ("cpu_int4", "cuda_int4"):
-        #     raise ValueError("Int4 optimization is not supported in phi2 optimum optimization")
+    pass_flows = [[]]
+    if args.finetune_method:
```

Comment:
chech whether it is qlora?

Reply:
Updated. But keep the finetune_method checking, and push the value of `finetune_method` to pass_flows. That is helpful once we introduce other new fine-tune way like Lora.